GerritStats/src/main/frontend/profile/PerMonthStatsPanel.jsx

```diff
@@ -34,33 +34,28 @@ export default class PerMonthStatsPanel extends React.Component {
         return elements;
     }
 
-    renderCommitCounts(commitTable, year) {
-        return this.getElementsPerMonth(commitTable, year, 'getPrintableMonthlyItemCount');
+    renderCounts(table, year) {
+        return this.getElementsPerMonth(table, year, 'getPrintableMonthlyItemCount');
     }
 
-    renderCommentCounts(commentTable, year) {
-        return this.getElementsPerMonth(commentTable, year, 'getPrintableMonthlyItemCount');
+    renderMoMChange(table, year) {
+        return this.getElementsPerMonth(table, year, 'getDisplayableMonthOnMonthChange');
     }
 
-    renderCommitsMoMChange(commitTable, year) {
-        return this.getElementsPerMonth(commitTable, year, 'getDisplayableMonthOnMonthChange');
-    }
-
-    renderCommitsQoQChange(commitTable, year) {
-        return this.getElementsPerMonth(commitTable, year, 'getDisplayableQuarterOnQuarterChange');
-    }
-
-    renderCommentsMoMChange(commentTable, year) {
-        return this.getElementsPerMonth(commentTable, year, 'getDisplayableMonthOnMonthChange');
-    }
-
-    renderCommentsQoQChange(commentTable, year) {
-        return this.getElementsPerMonth(commentTable, year, 'getDisplayableQuarterOnQuarterChange');
+    renderQoQChange(table, year) {
+        return this.getElementsPerMonth(table, year, 'getDisplayableQuarterOnQuarterChange');
     }
 
     renderPerYearStats() {
         const commitTable = this.props.userdata.datedCommitTable;
-        const years = this.props.userdata.datedCommitTable.getActiveYears();
+        const commentTable = this.props.userdata.datedCommentTable;
+        var years = commitTable.getActiveYears().concat(commentTable.getActiveYears());
+        years.sort(function(l, r) {
+            return (l > r) ? -1 : ((l < r) ? 1 : 0);
```

Comment:
I think you took this from the existing code, but could probably just do `l - r`.